### ISSUE TITLE
`convert_kps` may have missed some considerations when  `src==dst`

### ISSUE THREAD

**Issue by JasonBoy1** [open-mmlab/mmhuman3d#181]
When `src==dst` in [convert_kps](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/conventions/keypoints_mapping/__init__.py#L98-L102), it is easy to directly return the original `keypoints` and `all ones mask`. While there may have one circumstance that original mask is not `None`.
Here is my suggestions:
```python
    assert keypoints.ndim in {3, 4}

    src_names = keypoints_factory[src.lower()]
    dst_names = keypoints_factory[dst.lower()]
    if src == dst:
        if return_mask:
            if mask is None:
                if isinstance(keypoints, torch.Tensor):
                    return keypoints, torch.ones((len(dst_names)),
                                                 dtype=torch.uint8,
                                                 device=keypoints.device)
                elif isinstance(keypoints, np.ndarray):
                    return keypoints, np.ones((len(dst_names)), dtype=np.uint8)
            else:
                return keypoints, mask.reshape(-1)
        else:
            return keypoints
```
